examples/http_headers/main.go

```diff
@@ -71,7 +114,22 @@ func (ctx *httpHeaders) OnHttpRequestHeaders(numHeaders int, endOfStream bool) t
 }
 
 // Override types.DefaultHttpContext.
-func (ctx *httpHeaders) OnHttpResponseHeaders(numHeaders int, endOfStream bool) types.Action {
+func (ctx *httpHeaders) OnHttpResponseHeaders(_ int, _ bool) types.Action {
```

Comment:
please do not hide `numHeaders` and `endOfStream` as it's intentional to show users what's coming to the callbacks